### ISSUE TITLE
Error throwed when linting webpack project

### ISSUE THREAD

**Issue by jalooc** [charlierudolph/dependency-lint#11]
This was output when trying to lint React.js project built with webpack (thus using a lot of `import` even for resources like `scss` files).

```
C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\detective\node_modules\acorn\dist\acorn.js:1747
  throw err;
        ^
SyntaxError: 'import' and 'export' may appear only with 'sourceType: module' (2:0)
  at Parser.pp.raise (C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\detective\node_modules\acorn\dist\acorn.js:1745:13)
  at Parser.pp.parseStatement (C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\detective\node_modules\acorn\dist\acorn.js:2450:34)
  at Parser.pp.parseTopLevel (C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\detective\node_modules\acorn\dist\acorn.js:2379:21)
  at Object.parse (C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\detective\node_modules\acorn\dist\acorn.js:101:12)
  at parse (C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\detective\index.js:10:18)
  at Function.exports.find (C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\detective\index.js:45:15)
  at module.exports (C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\detective\index.js:24:20)
  at RequiredModuleFinder.findInContent (C:\****\AppData\Roaming\npm\node_modules\dependency-lint\src\linter\used_module_finder\required_module_finder.coffee:53:19)
  at C:\****\AppData\Roaming\npm\node_modules\dependency-lint\src\linter\used_module_finder\required_module_finder.coffee:32:21
  at fn (C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\async\lib\async.js:717:34)
  at C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\async\lib\async.js:1170:16
  at C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\async\lib\async.js:172:37
  at C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\async\lib\async.js:677:43
  at C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\async\lib\async.js:173:37
  at Object._onImmediate (C:\****\AppData\Roaming\npm\node_modules\dependency-lint\node_modules\async\lib\async.js:1163:34)
  at processImmediate [as _immediateCallback] (timers.js:345:15)
```

**Comment by charlierudolph** [comment]
Sorry for my late response. For this use case could the built file just be ignored? (with the config option `ignoreFilePatterns`)

**Comment by js08** [comment]
@charlierudolph so can you close it

**Comment by jalooc** [comment]
@charlierudolph what do you mean by ignoring the built file?

**Comment by charlierudolph** [comment]
Sorry, I may be mistaken. I was assuming this error was thrown when linting the webpack-ed file (which I was referring to as the built file). Do you know what file was being linted when this error was thrown? Can you please provide me with an example or way to reproduce this issue?

**Comment by jalooc** [comment]
@charlierudolph right now I don't have access to the repository, I'll try to exclude the built webpack file and find the source. Although the syntax error text seems to complain about including something like scss files from what I remember now (since they technically are not modules, and that's exactly what the package doesn't like).

**Comment by charlierudolph** [comment]
Just released 2.3.0 so the error message will include what filename of the file that was being parsed.

**Comment by charlierudolph** [comment]
Closing. Please reopen if needed
